Source/Core/InputCommon/ControllerInterface/OSX/OSXJoystick.mm

```diff
@@ -164,7 +164,7 @@
 {
   std::ostringstream s;
   s << IOHIDElementGetUsage(m_element);
-  return std::string("Button ") + StripSpaces(s.str());
+  return std::string("Button ") + std::string(StripSpaces(s.str()));
```

Comment:
Is `std::string + std::string_view` invalid?

Reply:
Yes. No such overload exists

Reply:
Maybe a `const char* + std::string_view` overload exists too? If not, I like the append version too.

Reply:
No, that doesn't exist either. (`const char*` is just a pointer and doesn't have any special overloads.) I'll use append here.